### PR TITLE
Issue 292 : Add schema and instances for body system (BodySystem)

### PR DESCRIPTION
**Pull request by ehennestad**

### CODE REVIEW

File: instances/bodySystem/cardiovascularSystem.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/cardiovascularSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "'Cardiovascular system' is an anatomical system that has as its parts the heart and blood vessels.",
```
Comment (UlrikeS91) [inline instances/bodySystem/cardiovascularSystem.jsonld]: "'The 'cardiovascular system' is an anatomical organ system where the heart pumps blood through blood vessels to and from all part of the body."

File: instances/bodySystem/centralNervousSystem.jsonld [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/centralNervousSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "In vertebrates 'central nervous system' consists of the neural tube derivatives: the brain and spinal cord. In invertebrates it includes central ganglia plus nerve cord.",
+  "description": "The central nervous system is the core nervous system that serves an integrating and coordinating function.",
```
Comment (UlrikeS91) [inline instances/bodySystem/centralNervousSystem.jsonld]: Better to flip them around:

definition: "The 'central nervous system' is the main processing center in most organisms. It's function is to take in sensory information, process information, and send out motor signals."

description: "In vertebrates, the central nervous system (CNS) consists of the brain and the spinal cord. In invertebrates, it includes the central ganglia and the nerve cord."

File: instances/bodySystem/digestiveSystem.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/digestiveSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "'Digestive system' is an anatomical system that has as its parts the organs devoted to the ingestion, digestion, and assimilation of food and the discharge of residual wastes.",
```
Comment (UlrikeS91) [inline instances/bodySystem/digestiveSystem.jsonld]: "The 'digestive system' is an anatomical organ system is composed of organs devoted to the ingestion, digestion, the assimilation of food and the discharge of residual wastes."

File: instances/bodySystem/musculoskeletalSystem.jsonld [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/musculoskeletalSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": null,
+  "description": "Anatomical system that consists of the muscular and skeletal systems.",
```
Comment (UlrikeS91) [inline instances/bodySystem/musculoskeletalSystem.jsonld]: definition: "The 'musculoskeletal system' is an anatomical organ system composed of organs providing the body with movement, stability, shape and support."

description: "The musculoskeletal system (sometimes also called locomotor system) is subdivided into two broader systems, the skeletal system and the muscular system. The skeletal system includes bones and joints. The muscular system includes all muscles in the body."

File: instances/bodySystem/vascularSystem.jsonld [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/vascularSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "'Vascular system' is an anatomical system that consists of all blood and lymph vessels.",
```
Comment (UlrikeS91) [inline instances/bodySystem/vascularSystem.jsonld]: "The 'vascular system' is an anatomical organ system that consists of all vessels in the body, and carries blood and lymph through all parts of the body."

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
As a general note, we always prioritize the definition over the description and the definition should be as short as possible (pref. only one single sentence).

**Comment by ehennestad** [comment]
Great suggestions Ulrike. I made the changes in this commit 92e4bfdf9a9f19351b4ea396f20d59dd0fd5bdab

File: instances/bodySystem/centralNervousSystem.jsonld [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/centralNervousSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "The 'central nervous system' is the main processing center in most organisms. It's function is to take in sensory information, process information, and send out motor signals.",
```
Comment (UlrikeS91) [inline instances/bodySystem/centralNervousSystem.jsonld]: "Its function..." (without the ')

File: instances/bodySystem/digestiveSystem.jsonld [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/digestiveSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "The 'digestive system' is an anatomical organ system is composed of organs devoted to the ingestion, digestion, the assimilation of food and the discharge of residual wastes.",
```
Comment (UlrikeS91) [inline instances/bodySystem/digestiveSystem.jsonld]: "The 'digestive system' is an anatomical organ system composed..." (not "system IS composed")

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

**Comment by lzehl** [comment]
For consistency we should probably redo this one and change the Schema to OrganismSystem (to match OrganismSubstance; which is also not fixed yet)

**Pull request merged**